justfile

```diff
@@ -0,0 +1,242 @@
+# Adventure Simulator - local development
+# Install just: cargo install just
+# List commands: just --list
+
+set shell := ["bash", "-eu", "-o", "pipefail", "-c"]
+
+spacetime_port := "3000"
+ui_port := "8000"
+tactical_port := "6000"
+public_bind := "0.0.0.0"
+
+spacetime_url := "http://localhost:" + spacetime_port
+spacetime_module := "strategic-stdb-module"
+
+strategic_dir := "crates/strategic-server/strategic-stdb-module"
+strategic_static := strategic_dir + "/static"
+
+run_dir := "/tmp/adventure-simulator-1"
+http_pid := run_dir + "/http.pid"
+http_log := run_dir + "/http.log"
+stdb_pid := run_dir + "/spacetime.pid"
+stdb_log := run_dir + "/spacetime.log"
+
+cert_dir := "utils"
+cert_pem := cert_dir + "/cert.pem"
+cert_key := cert_dir + "/key.pem"
+
+# Default recipe - show available commands
+default:
+    @just --list
+
+# Verify required tools are available
+preflight:
+    @command -v spacetime >/dev/null 2>&1 || { echo "Missing 'spacetime' CLI. Install it before running."; exit 1; }
+    @command -v python3 >/dev/null 2>&1 || { echo "Missing python3. Install it before running."; exit 1; }
```

Comment:
We can also do [`require`](https://just.systems/man/en/functions.html?highlight=find#executables) for executables